index.bs

```diff
@@ -12945,40 +12978,28 @@ the result of concatenating the [=identifier=] of the [=interface=] and the stri
 
 <h4 id="es-maplike">Maplike declarations</h4>
 
-Any object that [=implements=] an [=interface=]
-that has a [=maplike declaration=]
-must have a \[[BackingMap]] [=internal slot=], which is
-initially set to a newly created {{ECMAScript/Map}} object.
-This {{ECMAScript/Map}} object’s \[[MapData]] internal slot is
-the object’s [=map entries=].
-
 If an [=interface=] |A| is declared with
 a [=maplike declaration=], then
 there exists a number of additional properties on |A|’s
 [=interface prototype object=].
 These additional properties are described in the sub-sections below.
 
 <div algorithm>
+    To <dfn local-lt="coherency-check">coherency-check an object</dfn> |this|
```

Comment:
The algorithm name doesn't give me a lot of insight into what this is doing. Also, it says it's checking an object but the first step is ToObject. Maybe "security-and-implements check"?